src/main/java/com/lambdaschool/tiemendo/SeedData.java

```diff
@@ -18,14 +19,22 @@
     RetailerRepository retailerRepository;
     RetailerContactRepository retailerContactRepository;
     RetailerLocationRepository retailerLocationRepository;
+    OrganizationRepository organizationRepository;
+    OrganizationContactRepository organizationContactRepository;
+    OrganizationLocationRepository organizationLocationRepository;
```

Comment:
Nothing to change but just curious after our conversation about using abstract classes.
do you think we could do contact and location as their own models and have them connect to an abstract class?